lib/improver/standardise.py

```diff
@@ -195,9 +196,13 @@ def _regrid_to_target(self, cube, target_grid):
 
         attributes_to_inherit = (
             {key: val for (key, val) in target_grid.attributes.items()
-             if key in self.grid_attributes})
+             if key in MOSG_GRID_ATTRIBUTES})
```

Comment:
As this stands another organisation has no way of setting the attributes they wish to preserve without editing the code. That leads to forks, which means keeping everything up-to-date across contributing organisations becomes harder and harder.

Reply:
It's worth noting that here they are preserved, and at the spot step they are removed. I'm not sure other users would want exactly this behaviour. The configuration information may not be straight forward.

Reply:
No functional change here - simply centralised the list of grid attributes rather than setting them in `__init__`.

Reply:
Ah in terms of behaviour: this is exactly what the user should expect.  Because:

- In regridding we are gridding to a new grid.  So we want to take the grid information for the target grid and put it on our regridded field.
- In spot extraction we are no longer on a grid.  So we remove the grid information.

What other way would users expect this to behave?

Reply:
Re grid attributes: see comment above on CF standards.  This is also not about attribute preservation, but inheritance of CHANGED attributes from the target grid.

Reply:
However they are only inherited if they have MO names. Happy for this discussion around MO specific attributes to be followed up in a piece for follow on work and to proceed with what we have for now.